lib/options.ts

```diff
@@ -145,6 +145,7 @@ export function buildOptions(args: OktaAuthOptions = {}): OktaAuthOptions {
     devMode: !!args.devMode,
     storageManager: args.storageManager,
     cookies: isBrowser() ? getCookieSettings(args, isHTTPS()) : args.cookies,
+    tokenManager: args.tokenManager,
```

Comment:
If we use `this.tokenManager.getOptions` instead of `this.options.tokenManager`, this line is not necessary

Reply:
Do we still need this line? Actually let's not adding this line, and only use `this.tokenManager.getOptions` as the source of truth.

Reply:
Removed